source/blog/2018-09-07-week-5.html.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: blog-code
+title: Week 5
+author: Ian Drysdale
+---
+
+## The short version
+
+This week we:
+
+* got Fluidkeys to make a revocation certificate for your key and back it up
+* refreshed our homepage showing our planned roadmap
+* presented at the Liverpool Linux User Group meetup
+* decided 0.2 will allow you to setup key rotation
+
+## Fluidkeys now automatically creates and backs up revocation certificates when creating a key
+
+Two years ago I created a PGP Key, pushed it to a key server and then forgot about it. I also forgot my password that I used to secure it 🤦‍ This basically renders the key useless. Worse still, it didn't have an expiry date! Fortunately, PGP has a way you can mark such a dud key as `revoked`. The catch is that in order to do this, you need to have generated a revocation certificate... which is impossible if you've forgotten your password! 😡
+
+We figured it'd be sensible for Fluidkeys to automatically create this revocation certificate for you when you create a key and back it up for you. That way, if you do ever run in to problems with they key it's easy to mark it as revoked.
+
+## Fluidkeys roadmap is now online
+
+We've refreshed our homepage at www.fluidkeys.com. On it we now have a checklist of features working towards our 1.0 release. We also decided to make our Trello board (of prioritised tasks) public. If you _really_ want to snoop on us, this is the detail 🔎
+
+## Paul presented at Liverpool Linux User Group (LivLUG)
+
+On Wednesday evening, Paul presented at the monthly Liverpool Linux User Group. He spoke about the superpowers that PGP can give people, but the challenges they face in using it. We also demoed Fluidkeys. Thanks to everyone who came along we had a great discussion afterwards. We were looking for connections to sysadmins that might want to test it and for introductions to open source entrepreneurs who'd successfully built businesses around their software.
+
+## Fluidkeys 0.2 will automatically rotate your keys each month
+
+We've got Fluidkeys generating new keys, or linking to keys you've already made in GPG. Now time for something more useful: key rotation!
+
+Key rotation is what it sounds like, tossing out an old one, and replacing it with a new one. Given how complicated it is to create a key using GPG in the first place, it's no suprise that the majority of keys out there aren't set to expire, nor rotate. But rotating your keys is considered good practice:
```

Comment:
The 3 sentences here together make it sound like we're going to be tossing out the *whole key*, which is not desirable. The logic of "it's hard to create a key in the first place" -> "people don't rotate keys" isn't right... because rotation will be done via *subkeys*. 

My suggestion would be something like this:

> Key rotation is what it sounds like, tossing out an old one, and replacing it with a new one. OpenPGP allows you to do key rotation without having to make a whole new key, but it involves fiddly subkeys and in practice very few people do it.

Reply:
Thanks good shout!